zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -3818,21 +3818,21 @@ components:
           type: integer
           format: int64
           description: The key of the process instance.
-        decisionKey:
+        decisionDefinitionKey:
           type: integer
           format: int64
           description: The key of the decision.
-        dmnDecisionId:
+        decisionDefinitionId:
           type: string
           description: The ID of the DMN decision.
-        dmnDecisionName:
+        decisionDefinitionName:
           type: string
           description: The name of the DMN decision.
-        decisionVersion:
+        decisionDefinitionVersion:
           type: integer
           format: int32
           description: The version of the decision.
-        decisionType:
+        decisionDefinitionType:
           $ref: '#/components/schemas/DecisionInstanceTypeEnum'
```

Comment:
Is this a decision INSTANCE type or decision DEFINITION type? @ralfpuchert, any input here?

Reply:
Waiting for inputs

Reply:
yes this should be a `DecisionDefinitionType`